src/main/launcher/options.c

```diff
@@ -19,6 +19,7 @@ void options_init(struct options *options)
     options->app_config_path = "prop/app-config.xml";
     options->avs_config_path = "prop/avs-config.xml";
     options->ea3_config_path = "prop/ea3-config.xml";
+    options->ea3_ident_path = "prop/ea3-ident.xml";
```

Comment:
Default is ok, but I don't see any parsing for this in `options_read_cmdline` allowing users to set this one to a path (and file) of their choice.